mscore/importmxmlpass1.cpp

```diff
@@ -542,13 +542,13 @@ static void doCredits(Score* score, const CreditWordsList& credits, const int pa
              pf->width(), pf->height(), pf->oddTopMargin(), score->spatium(), DPMM, DPI);
       */
       // page width, height and odd top margin in tenths
-      const double ph  = score->styleD(Sid::pageHeight) * 10 * DPI / score->spatium();
+      const double ph = score->style().pageOdd().heightPoints() * SPATIUM_XML / score->spatium();
```

Comment:
Today I replaced all the relevant instances of `10` and `10.0` with my new constant `SPATIUM_XML`.  In the process I realized that this module could probably use a reverse version of the same refactoring I did in exportxml.cpp.  I will look into this, but it might be a separate PR.